blocks/spinning-cards/spinning-cards.css

```diff
@@ -0,0 +1,72 @@
+.spinning-cards-grid  {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 30px;
+  margin: 0;
+  padding: 0;
+}
+
+.spinning-cards-item {
+  width: auto;
+  height: 250px;
+  perspective: 1000px;
+  float:left;
+  list-style: none;
+  margin-top: 20px;
+}
+
+.spinning-cards-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  transition: transform 0.6s;
+  transform-style: preserve-3d;
+  border: 1px solid  var(--c-dark-plum);
+}
+
+.spinning-cards-item:hover .spinning-cards-inner {
+  transform: rotateY(180deg);
+}
+
+.spinning-cards-front, .spinning-cards-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  backface-visibility: hidden;
+}
+
+.spinning-cards-front {
+  background-color: var(--c-dark-plum);
+  color: var(--c-white);
+  border: 1px solid  var(--c-dark-plum);
+}
+
+.spinning-cards-front  h4 {
+  color: var(--c-white);
+  line-height: 0.2;
+}
+
+.spinning-cards-back {
+  color: var(--c-dark-plum);
+  transform: rotateY(180deg);
+  font-size: 14px;
+  padding-top: 65px;
```

Comment:
![Bildschirmfoto 2023-07-13 um 09 11 50](https://github.com/Netcentric/cn-website/assets/135241880/9579b404-5357-4e45-826a-1ed7d24feb23)
To avoid this layout issue you need to set also a padding to left and right. For example 10px. And to keep everything in place you need also do define the `box-sizing` to `border-box`

I have also checked the specification, for me it is not clear if the text should always start 65px from top. This is what you achieved with the `padding-top: 65px`. But if the text should be vertical aligned center I would go with a `flex` approach.